config/environments/development.rb

```diff
@@ -38,4 +38,12 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_credentials: {
+      bucket: "cardapp",
+      access_key_id: ENV['s3_key'],
+      secret_access_key: ENV['s3_secret']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.